packages/grafana-ui/src/components/DataSourceSettings/DataSourceHttpSettings.tsx

```diff
@@ -78,9 +78,31 @@ export const DataSourceHttpSettings = (props: HttpSettingsProps) => {
   } = props;
 
   const [isAccessHelpVisible, setIsAccessHelpVisible] = useState(false);
+  const [azureAuthEnabled, setAzureAuthEnabled] = useState(false);
   const theme = useTheme2();
   let urlTooltip;
 
+  useEffect(() => {
```

Comment:
Rather than an effect, can you do this in `onSettingsChange` ?

Reply:
@joshhunt I'll make the change. Not sure if that's going to be the one you asked for.

Reply:
@joshhunt Please see https://github.com/grafana/grafana/pull/72472/commits/3695109a4453bc4d5c2d7d02cf32d6ad79178c61